src/main/java/net/sacredlabyrinth/phaed/simpleclans/hooks/discord/DiscordHook.java

```diff
@@ -471,18 +502,22 @@ private void resetPermissions() {
                 filter(Objects::nonNull).
                 map(Clan::getMembers).
                 flatMap(Collection::stream).
-                forEach(clanPlayer -> updatePermissions(clanPlayer, clanPlayer.getClan(), ADD));
+                forEach(clanPlayer -> updateViewPermission(clanPlayer, ADD));
     }
 
     private void createChannels() {
-        clanTags.removeAll(discordClanTags);
         for (String clan : clanTags) {
             try {
                 createChannel(clan);
             } catch (CategoriesLimitException | ChannelsLimitException ex) {
```

Comment:
If the limit is reached, the loop will break here and not process more of the invalid channel logic.